library/init/coe.lean

```diff
@@ -4,7 +4,8 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Leonardo de Moura
 -/
 
-/-
+/-! # Coercions and lifts
```

Comment:
I think the `prelude` and `import` lines need to be moved above this module docstring.